rcsbapi/const.py

```diff
@@ -27,3 +37,34 @@ class Const(Enum):
     RCSB_SEARCH_API_QUERY_URL: str = "https://search.rcsb.org/rcsbsearch/v2/query"
     UPLOAD_URL: str = "https://user-upload.rcsb.org/v1/putMultipart"
     RETURN_UP_URL: str = "https://user-upload.rcsb.org/v1/download/"
+    #
+    # Data API constants
+    DATA_API_ENDPOINT: str = "https://data.rcsb.org/graphql"
```

Comment:
Should DATA_API_ENDPOINT be moved to config in response to Yana's feedback about being able to change the endpoint?